sample/shared/build.gradle.kts

```diff
@@ -94,22 +94,22 @@ kotlin {
 
     targets.withType<KotlinNativeTarget> {
         binaries.withType<Framework> {
-            export("co.touchlab:kermit")
+            export("co.touchlab:kermit:0.3.1-m1")
             transitiveExport = true
         }
     }
 }
 
-//android {
-//    compileSdkVersion(29)
-//    defaultConfig {
-//        minSdkVersion(15)
-//        targetSdkVersion(29)
-//        versionCode = 1
-//        versionName = "1.0"
-//    }
-//
-//    val main by sourceSets.getting {
-//        manifest.srcFile("src/androidMain/AndroidManifest.xml")
-//    }
-//}
+android {
+    compileSdkVersion(29)
```

Comment:
Looks like this sample is not using latest gradle version based on this syntax. We should update this sample to 7.0.2 too.

Reply:
updated gradle version